numpy/core/numeric.py

```diff
@@ -2276,6 +2276,9 @@ def isclose(a, b, rtol=1.e-5, atol=1.e-8, equal_nan=False):
     relative difference (`rtol` * abs(`b`)) and the absolute difference
     `atol` are added together to compare against the absolute difference
     between `a` and `b`.
+    
+    .. warning:: The default `atol` is not appropriate for comparing numbers
+                 that are much smaller than one (see Notes).
```

Comment:
Could we add that suggested/safe atol is 0.0, that should be tuned as needed?

Reply:
In general the warning should be as brief as possible. If you suggest phrasing for the notes, I would consider updating it.